cmd/application.go

```diff
@@ -22,8 +22,20 @@ var applicationCmd = &cobra.Command{
 	Use:     "app",
 	Short:   "Perform application operations",
 	Aliases: []string{"application"},
-	// 'odo application' is the same as 'odo application get'
-	Run: applicationGetCmd.Run,
+	Example: `  # Get current application,
+  odo app 
+  # Set webapp to current application,
```

Comment:
can you please add an empty line between each example?
It would be nice to keep it consistent with other places like `odo create -h`